### ISSUE TITLE
Allow for a gradient step schedule for modelbuilding

### ISSUE THREAD

**Issue by gdevenyi** [CoBrALab/optimized_antsMultivariateTemplateConstruction#5]
As suspected previously, 0.25 might be too large for final iterations so a step-size schedule may make more sense here.

https://github.com/ANTsX/ANTs/issues/1172#issuecomment-835327157

**Comment by gdevenyi** [comment]
Related to #2

**Comment by gdevenyi** [comment]
Adjust --gradient-step option to either be a single value, or arbitrary list
```python
>  np.geomspace(0.5,0.05,4)
array([0.5       , 0.23207944, 0.10772173, 0.05      ])
```

**Comment by gdevenyi** [comment]
Fixed in #29
